tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -449,6 +449,19 @@ protected void EnableDirectLogSubmission(int intakePort, string integrationName,
             SetEnvironmentVariable(ConfigurationKeys.ApiKey, "DUMMY_KEY_REQUIRED_FOR_DIRECT_SUBMISSION");
         }
 
+        protected void EnableTelemetry(bool enabled = true, int? standaloneAgentPort = null)
```

Comment:
Small suggestion: Maybe you could refactor this so the second parameter is just a string which represents the agentless telemetry URL. It could make the helper method more simple, at the cost of making it a little more hassle for the caller. Up to you, just a small thought I had 🤷🏼

Reply:
I'm not quite following how that makes the method simpler🤔 AFAICT, we would still would need all the same logic, we'd just be working with a `string` instead of an `int?`?

My goal (in a subsequent PR, once it's enabled by default), is to make enabling telemetry the default in part of the agent, at which point, this method should only rarely be needed (only in tests where we explicitly want to test the "agentless" path), at which point it might be ripe for a refactoring 🙂

Reply:
PR for that is here: https://github.com/DataDog/dd-trace-dotnet/pull/2800